test/topics.js

```diff
@@ -81,6 +81,21 @@ describe('Topic\'s', () => {
             });
         });
 
+        it('should create a new private topic', (done) => {
+            topics.post({
+                uid: topic.userId,
+                title: topic.title,
+                content: topic.content,
+                cid: topic.categoryId,
+                privateTopic: true,
+            }, (err, result) => {
+                assert.ifError(err);
+                assert(result);
+                topic.tid = result.topicData.tid;
```

Comment:
might be a good idea to add something along the lines of `assert.equal(result.topicData.privateTopic, true);` to make sure it actually set it as true here

Reply:
done-zo